conf/rh_cloud.yaml.template

```
@@ -2,3 +2,4 @@ RH_CLOUD:
   INSTALL_RHC: false
   ORGANIZATION: org_name
   ACTIVATION_KEY: ak_name
+  CRC_ENV: "prod"
```

As there are only two possible env's rn, prod, and stage, so we can also make this a boolean, and this will require an updated on satqe-jenkins repo too, non-blocking :)
```suggestion
  CRC_PROD: True
```

---

We might need an extra calculation in CI to set this to True/False when running the job instead of current passing fwd whatever we have set in the parameter.

---

Thanks for the suggestion but I prefer the existing approach.